### PR TITLE
1513939: Add android reference browser

### PR DESCRIPTION
**Pull request by mdboom**
This adds a new application url `android-reference-browser`, which will take vanilla glean telemetry.

Also, this adds the parquet template to `glean`, which was missed in #230.

Question (@fbertsch, @mreid): How do we want to handle testing?  It seems redundant / potentially hard to keep in sync to just copy the tests from `glean` into `android-reference-browser`, but it doesn't appear that the templating works on test content.  Any thoughts?

Checklist for reviewer:

- [x] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs
- [ ] Trigger the `integration` CI test by pushing this revision [as discussed in the README](https://github.com/mozilla-services/mozilla-pipeline-schemas#packaging-and-integration-tests-optional)

### CODE REVIEW

**Comment by mreid-moz** [comment]
Agreed that it's redundant to copy the tests over. I think we should include an actual example of a ping produced by the new application `android-reference-browser`, and (if applicable) test pings for anything that is unique/different about the new application.

**Comment by mdboom** [comment]
So in that case, I suppose that means no tests for now, since the new application isn't integrated with glean yet and we're not aware of any application-specific differences for now...

**Comment by fbertsch** [comment]
Also not really any integration tests we can do, since there's no data available.

File: schemas/glean/events/events.1.parquetmr.txt [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+      repeated group key_value {
+        required binary key (UTF8);
+        required binary value (UTF8);
+      }
+    }
+  }
+  optional group events (LIST) {
+    repeated group list {
+      required group element (TUPLE) {
+  required int64 timestamp;
```
Comment (fbertsch) [inline schemas/glean/events/events.1.parquetmr.txt]: These issues should have been fixed with the recent parquetfmt changes, can you rebase and rebuild the schemas?

Comment (mdboom) [reply schemas/glean/events/events.1.parquetmr.txt]: Sure thing.  Should be done now.

**Comment by mdboom** [comment]
@fbertsch: I think I got this now.  We'll have to convert `org.mozilla.reference.browser` to `org-mozilla-reference-browser` on the clients end before hitting this URL.

**Pull request merged**